src/components/ui/OakPrimaryInvertedButton/OakPrimaryInvertedButton.stories.tsx

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import { Meta, StoryObj } from "@storybook/react";
+
+import { OakPrimaryInvertedButton } from "./OakPrimaryInvertedButton";
+
+import { oakIconNames } from "@/components/base/OakIcon";
+import { OakFlex } from "@/components/base";
+
+const controlIconNames = [null, [...oakIconNames].sort()].flat();
+
+/**
+ *
+ * A specific implementation of InternalRectButton
+ *
+ * The following callbacks are available for tracking focus events:
+ *
+ * ### onClick
+ * `onClick?: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;`
+ *
+ * ### onHovered
+ *  `onHovered?: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>, duration: number) => void;`<br>
+ *  called after a mouseEnter and mouseLeave event has happened
+ */
```

Comment:
If this doc block is moved above the component itself here: https://github.com/oaknational/oak-components/blob/ce6f7e32f485dfc147c61651cbae7d7513070e00/src/components/ui/OakPrimaryInvertedButton/OakPrimaryInvertedButton.tsx#L22

then will show up in both Storybook and as a typescript hint in VSCode when using the component. Really helpful when consuming it in OWA.

We have these blocks in the stories in some places but i've been making this change with new components and want to do the same with older components.

Reply:
good shout, updated